pyproject.toml

```
@@ -81,7 +81,7 @@ documentation = "https://docs.saleor.io/"
   Rx = "^1.6.3"
   semantic-version = "^2.10.0"
   sendgrid = "^6.7.1"
-  sentry-sdk = "^1.28"
+  sentry-sdk = "1.35"
```

A comment is always helpful for maintainers/next readers:

```suggestion
  sentry-sdk = "1.35" # https://github.com/getsentry/sentry-python/issues/2554
```